### PR TITLE
cleanup(clowder): RHICOMPL-1978 mark SECRET_KEY_BASE as a placeholder

### PR DESCRIPTION
**Pull request by skateman**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: clowdapp.yaml [hunk 1]
```diff
@@ -66,9 +66,6 @@ objects:
           value: "${PATH_PREFIX}"
         - name: APP_NAME
           value: "${APP_NAME}"
-        # Should be a secret ?
-        - name: SECRET_KEY_BASE
```
Comment (vkrizan) [inline clowdapp.yaml:70]: Wouldn't Rails shout when this is not defined?

Comment (skateman) [reply clowdapp.yaml:70]: Hmm, it might be :see_no_evil: I'll check next week

Comment (skateman) [reply clowdapp.yaml:70]: Yeah, we'll just mark this as a placeholder in the clowdapp :+1:

**Review by vkrizan — APPROVED** [review]

**Pull request merged**